internal/cmd/policy/sign/sign.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package sign
+
+import (
+	"os"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/policy"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p          *persistent.Options
+	policyName string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.policyName,
+		"policy-name",
+		policy.TargetsRoleName,
+		"policy file to sign",
```

Comment:
(non-blocking): When I see "file" I think this should be a file path, but based on the default this is really a target name? We may want to be more intentional about naming, or better define what we mean by "policy file" 🤔

Reply:
I'll address all instances of this in a separate PR after this one, thanks for pointing it out.